Integrations/ML/Lenses/Archaeologist/ModLens_Archaeologist.lua

```diff
@@ -0,0 +1,156 @@
+local LENS_NAME = "ML_ARCHAEOLOGIST"
```

Comment:
this is annoying - I did the proper Git move command.  Git doesn't like the move + changes, in this case, it seems?

![image](https://user-images.githubusercontent.com/8787640/93006275-08d96180-f50f-11ea-8d6b-21030979139b.png)